packages/dx-react-chart/docs/guides/series.md

```diff
@@ -0,0 +1,82 @@
+# React Chart - Series
+
+A series is a collection of related data points. The Chart provides 6 plugins for each type of series. All plugins have 'pointComponent' that renders points. LineSeries, SplineSeries and AreaSeries plugins have also 'pathComponent' that renders line or area path connected their points.
+
+## Line and Spline Series
+
+The LineSeries plugin visualize data as a collection of points connected by a line.
+
+<div
+    class="embedded-demo"
+    data-options='{"path":"/demo/chart-basic/line.skip","showThemeSelector":true,"scriptPath":"{{site.baseurl}}/{{page.demos_link}}/demos/dist/index.js?v={{ site.time | date: '%s' }}"}'
+>
+</div>
+
+The SplineSeries plugin visualize data the same way as the LineSeries plugin only it draws curved line.
```

Comment:
Usually, we use the \` symbol when pointing a plugin name.

For example:

The \`TableHeaderRow\` plugin...

=>

![image](https://user-images.githubusercontent.com/1897913/39925901-2afcf2e8-5536-11e8-98c3-65162b4c0398.png)